.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,15 @@
 ## Description
-
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes in detail. The sections suggested below  -->
 
 ## Motivation / Context
 <!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes, or is related to, an open issue, link to the issue here. -->
+<!--- If it fixes, or is related to an open issue, link to the issue here. -->
```

Comment:
What do you think about removing that first comma as well? I think it flows a bit better that way.
```suggestion
<!--- If it fixes or is related to an open issue, link to the issue here. -->
```